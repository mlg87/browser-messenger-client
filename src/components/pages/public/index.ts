export { default as Landing } from './Landing/Landing';
export { default as Login } from './Login/Login';
export { default as NotFound } from './NotFound/NotFound';
export { default as Register } from './Register/Register';
