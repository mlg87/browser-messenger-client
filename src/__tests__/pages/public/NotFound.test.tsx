import { render } from '@testing-library/react';
import React from 'react';
import { MemoryRouter } from 'react-router';
import { NotFound } from '../../../components/pages/public';

const setup = () => render(
    <MemoryRouter>
        <NotFound location={{ pathname: '/nope' }} />
    </MemoryRouter>
);

describe('NotFound', () => {

    it('renders', () => {

        const { container } = setup();
        expect(container.firstChild).toMatchSnapshot();

    });

});
