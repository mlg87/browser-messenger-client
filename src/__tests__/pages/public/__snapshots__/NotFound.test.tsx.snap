// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders 1`] = `
<div>
  <h1>
    404: Not Found
  </h1>
  <p>
    Sorry, Charlie. We don't have anything at 
    <strong>
      /nope
    </strong>
  </p>
</div>
`;
